include/oneapi/dpl/dynamic_selection

```diff
@@ -19,6 +19,13 @@
     #define _DS_BACKEND_SYCL 0
 #endif
 
-#include "oneapi/dpl/internal/dynamic_selection.h"
+#include "oneapi/dpl/internal/dynamic_selection_traits.h"
 
+#if _DS_BACKEND_SYCL != 0
```

Comment:
include of backend should be removed. Each policy will include it as needed. This file should not.

Reply:
Fixed